addons/portal/models/mail_thread.py

```diff
@@ -35,7 +36,7 @@ def _notify_get_recipients_groups(self, message, model_description, msg_vals=Non
             local_msg_vals['access_token'] = access_token
             local_msg_vals['pid'] = customer.id
             local_msg_vals['hash'] = self._sign_token(customer.id)
-            local_msg_vals.update(customer.signup_get_auth_param()[customer.id])
+            local_msg_vals.update(customer.sudo().signup_get_auth_param()[customer.id])
```

Comment:
Why ? Seems more like a fix, if necessary ?

Reply:
We reduced usage of `sudo` in portal when users have read/write (according to message post rights).
Previously providing a token always used sudo, even when non-necessary.

This highlighted 2 or 3 occurrences of missing sudo in lower level code.

Reply:
Ok. Probably worth a comment in code ?